### PR TITLE
Fix issue where duplicate filenames were getting in middleware packages

### PR DESCRIPTION
**Pull request by bbowman**

### CODE REVIEW

File: tools/WinObjC.Packaging/WinObjC.Packageable.targets [hunk 1]
```diff
@@ -78,7 +78,8 @@
         <PackagePath>$(PackageResourcesBasePath)\%(Filename)%(Extension)</PackagePath>
       </PackageFile>
       <!-- Package up .pri files associated with any .winmds -->
-      <PackageFile Include="$(ProjectPriFullPath)" Condition="Exists('$(ProjectPriFullPath)') and Exists('@(WinMDFullPath)')">
+      <_PriFileNames Include="$(ProjectPriFullPath)" />
+      <PackageFile Include="@(_PriFileNames)" Condition="Exists('%(FullPath)') and Exists('$(_WinMDFilePath)')">
```
Comment (DHowett-MSFT) [inline tools/WinObjC.Packaging/WinObjC.Packageable.targets]: where did this underscored name come from?

Comment (DHowett-MSFT) [inline tools/WinObjC.Packaging/WinObjC.Packageable.targets]: it'll be worth documenting :smile:

Comment (bbowman) [inline tools/WinObjC.Packaging/WinObjC.Packageable.targets]: What do you mean? It's a private implementation detail. It's like a member variable.

Comment (DHowett-MSFT) [inline tools/WinObjC.Packaging/WinObjC.Packageable.targets]: I was talking about `_WinMDFilePath`; it didn't look like it was introduced in this pull request

**Pull request merged**